pkgs/desktops/elementary/apps/elementary-calculator/default.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv, fetchFromGitHub, elementary, pkgconfig
+, meson, ninja, vala, desktop-file-utils, libxml2
+, gtk3, python3, granite, libgee, gobject-introspection
+, defaultIconTheme, appstream, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "calculator";
+  version = "1.5.1";
+
+  name = "elementary-${pname}-${version}";
```

Comment:
Well I'm currently using `pname` for something that would be better called `repoName`.
Since I reuse it for the updateScript and for the repo name for fetchFromGitHub.

Reply:
Gah, GitHub’s hidden conversations are my nemesis. I think it might be a little confusing since since https://github.com/NixOS/nixpkgs/pull/49398, `pname` is used as the name part of derivation name. Maybe `appName` or `projectName` would be better names.

Reply:
IIRC there would be a trace if I were doing this, but I think that was reverted. 
I'll put one of those names outside the package description.
Then `pname` will be like `pname = "elementary-${someName}";`

Though I'm still not sure how they feel about us calling any of this "elementary".